tests/test_flake8_trio.py

```diff
@@ -207,11 +202,6 @@ def test_noerror_on_sync_code(test: str, path: str):
     )
 
 
-def read_file(test_file: str):
-    filename = Path(__file__).absolute().parent / test_file
-    return Plugin.from_filename(str(filename))
-
-
```

Comment:
With moving everything to using `Path` this function can now be completely removed :tada: